client/src/components/ProductDetailPage.jsx

```diff
@@ -26,8 +26,6 @@ class ProductDetailPage extends React.Component {
       currentProductId: 22126, // this should be set at runtime by the productId in the url? or if none given, has a default
       relatedProducts: [],
       products: {
-        // mock products until successful API calls
-        '22126': productsMock
```

Comment:
I see you are removing the default product. Is there handling for the exception in the components if the API fails?

Reply:
The problem is that the default product does not match the product we are pulling in from our API and makes the app crash.

Reply:
I guess we could fix the mock